src/styles/blocks/card.scss

```diff
@@ -0,0 +1,72 @@
+.card {
+  box-sizing: border-box;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  width: 200px;
+  padding: 32px 16px 16px;
+  display: flex;
+  flex-direction: column;
+  font-size: 10px;
+  color: $base-color;
```

Comment:
The variable `$base-color` is used here. Ensure that it is defined in a separate SCSS variables file and imported correctly.